tapgarden/planter.go

```diff
@@ -773,18 +835,14 @@ func (c *ChainPlanter) gardener() {
 	}
 }
 
-// finalizeBatch creates a new caretaker for the batch and starts it.
-func (c *ChainPlanter) finalizeBatch(params FinalizeParams) (*BatchCaretaker,
-	error) {
-
+// If funding fails should we delete the batch?
+func (c *ChainPlanter) fundBatch(ctx context.Context, params FundParams) error {
```

Comment:
Will this routine be called multiple times now? Otherwise why would the batch already be funded before this call?